.github/workflows/release.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       uses: goreleaser/goreleaser-action@v2
       with:
         args: release --rm-dist
-        version: v0.155.1
+        version: v0.173.2
```

Comment:
This change actually needs to be made in the source CUE, which is why CI is failing.

It's easily missed, but this file is generated, per the header:

https://github.com/cue-lang/cue/blob/7d25ee232fe7ee57c502baee7e82417a03a7e297/.github/workflows/release.yml#L1

So the change ends up needing to be made here:

https://github.com/cue-lang/cue/blob/7d25ee232fe7ee57c502baee7e82417a03a7e297/internal/ci/workflows.cue#L243

After making that change, you need to regenerate the workflow files:

```
go generate ./internal/ci
```

As if this process weren't involved enough already, we actually have tests that are based on our workflow setup (which drastically reduces the iteration time when working on changes to the core evaluation engine). Those tests include golden files that are a the result of generating Yaml from CUE (amongst a number of others). To update those golden files, run:

```
CUE_UPDATE=1 go test ./...
CUE_UPDATE=1 go test ./...
```

Note that I have deliberately run that command twice, above.

Then a plain:

```
go test ./...
```

should succeed, and:

```
go generate ./internal/ci
```

should at that point leave you with a clean commit.

Reply:
oh, totally missed that part... fixing!